src/main/java/se/sundsvall/notes/api/RevisionResource.java

```diff
@@ -0,0 +1,58 @@
+package se.sundsvall.notes.api;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.hibernate.validator.constraints.Range;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.zalando.problem.Problem;
+import org.zalando.problem.violations.ConstraintViolationProblem;
+import se.sundsvall.dept44.common.validators.annotation.ValidUuid;
+import se.sundsvall.notes.api.model.DifferenceResponse;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON_VALUE;
+import static org.springframework.http.ResponseEntity.ok;
+
+@RestController
+@Validated
+@RequestMapping("/notes")
+@Tag(name = "Revisions", description = "Operations for revisions")
+public class RevisionResource {
+
+	@GetMapping(path = "/{id}/revisions", produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE })
+	@Operation(summary = "Get revisions by note ID")
+	@ApiResponse(responseCode = "200", description = "Successful operation", content = @Content(mediaType = APPLICATION_JSON_VALUE, schema = @Schema(implementation = FindRevisionsResponse.class)))
```

Comment:
Instead of: 
`content = @Content(mediaType = APPLICATION_JSON_VALUE, schema = @Schema(implementation = FindRevisionsResponse.class)`

Can this be the following:
`content = @Content(mediaType = APPLICATION_JSON_VALUE, array = @ArraySchema(schema = @Schema(implementation = Revision.class)))`
...and remove FindRevisionResponse?

The FindRevisionsResponse is just a wrapper over a `List<Revision>`, right?